runtime/src/accounts_db.rs

```diff
@@ -4325,17 +4325,14 @@ impl AccountsDb {
                 continue; // skipping slot with no useful accounts to write
             }
 
-            let total_accounts_after_shrink = alive_accounts.len();
-
             let (_, time) = self.maybe_create_ancient_append_vec(&mut current_ancient, slot);
             let mut create_and_insert_store_elapsed = time.as_micros() as u64;
             let (ancient_slot, ancient_store) =
                 current_ancient.as_ref().map(|(a, b)| (*a, b)).unwrap();
             let available_bytes = ancient_store.accounts.remaining_bytes();
-            let mut start = Measure::start("find_alive_elapsed");
+            let mut find_alive_elapsed = Measure::start("find_alive_elapsed");
             let to_store = AccountsToStore::new(available_bytes, &alive_accounts, slot);
-            start.stop();
-            let find_alive_elapsed = start.as_us();
+            find_alive_elapsed.stop();
```

Comment:
just use measure!:
```
let (to_store, find_alive_elapsed) = measure!(AccountsToStore::new(available_bytes, &alive_accounts, slot));
```
feel free to add the "find_alive_elapsed" name, but we don't use it so also good to eliminate the clutter.